### ISSUE TITLE
Param regressor for MANO

### ISSUE THREAD

**Issue by pablovela5620** [mks0601/I2L-MeshNet_RELEASE#33]
I had a question regarding the following [ParamRegressor](https://github.com/mks0601/I2L-MeshNet_RELEASE/blob/7b9aabf93535deca01b988c9d6ef02f002ffe7c8/common/nets/module.py#L126) part of the code. 

First, why are you concatenating a zero tensor of shape (batch, 3, 1) to the 2nd axis? I see that you're converting from rotation matrix to angle axis, and you're using torch geometry (which is kornia now I think?). Is this just to convert the rotation matrix to a transformation matrix with 0 translation to be compatible with torch geometry? 

```python 
        pose = torch.cat([pose,torch.zeros((pose.shape[0],3,1)).cuda().float()],2)
        pose = tgm.rotation_matrix_to_angle_axis(pose).reshape(-1,72)
```

It also looks like the reshape to 72 only works for SMPL estimation and it should be 48 (16*3) for mano?

**Comment by mks0601** [comment]
`rotation_matrix_to_angle_axis` requires 3x4 rotation matrix.

**Comment by mks0601** [comment]
https://kornia.readthedocs.io/en/v0.1.2/conversions.html#torchgeometry.rotation_matrix_to_angle_axis

**Comment by pablovela5620** [comment]
I see, I was using a newer version of kornia that changed that to a 3x3 so that was my confusion
https://kornia.readthedocs.io/en/latest/geometry.conversions.html?highlight=rotation#kornia.geometry.conversions.angle_axis_to_rotation_matrix
